components/fxa-client/Cargo.toml

```diff
@@ -20,6 +20,7 @@ sync15 = { path = "../sync15" }
 url = "2.1"
 ffi-support = "0.4"
 viaduct = { path = "../viaduct" }
+restmail-client = { path = "../support/restmail-client" }
```

Comment:
I don't think it's going to be a big deal in practice, but part of me feels bad about including code here that's only going to be used for tests, but will still take up space in the compiled library, the Fenix app, etc.

I don't have a good suggestion for what to do about it though :-(

Reply:
Agree, for what it's worth it's a very small crate (<100 LOC)